utils/soca/gdas_soca_diagb.h

```diff
@@ -495,13 +503,24 @@ namespace gdasapp {
       const eckit::LocalConfiguration bkgErrorConfig(fullConfig, "background error");
       soca::Increment bkgErrOut(geomOut, bkgErr);
       bkgErrOut.write(bkgErrorConfig);
-
+      oops::Log::info() << "====================== Background Error:" << std::endl;
+      oops::Log::info() << bkgErrOut << std::endl;
       return 0;
     }
 
     // -----------------------------------------------------------------------------
 
    private:
+    // Function to compute the local e-folding scale
+    double computeLocalEFoldingScale(const double depth, const double eFoldingLength) const {
+      const double minRatio = 3.0;
```

Comment:
should this be `edRatio`?

Reply:
:man_facepalming: Yes ... 
Sorry, looks like somebody forgot to double check before creating this PR.